### PR TITLE
eoc refactoring

### PR DESCRIPTION
**Pull request by wilkus27**
Epic: https://github.com/openstax/cnx-recipes/issues/4683

### CODE REVIEW

File: styles/designs/cosmos/parts/_keyterms-components.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    _subselector: " > dl",
+    _properties:(
+        text-indent: h-spacing(-3),
+        padding-left: h-spacing(3),
+    ),
+);
+
+$KeyTerms__Term:(
+    _name: "KeyTermsTerm",
+    _subselector: " > dt ",
```
Comment (ajluk) [inline styles/designs/cosmos/parts/_keyterms-components.scss:20]: @wilkus27 It is small thing - there is additional space added at the end of ` > dt` selector. Same thing in the below `KeyTermsDefinition` component.

File: styles/output/statistics-pdf.css [hunk 2]
```diff
@@ -1146,6 +1146,13 @@ a[role=doc-noteref] {
   content: target-counter(attr(href, url), page);
 }
 
+.os-formula-review-container {
+  margin-bottom: 1.4rem;
```
Comment (ajluk) [inline styles/output/statistics-pdf.css:1150]: I have noticed that here and in line 2960 from below this property is repeated.

**Review by ajluk — COMMENTED** [review]
I have only noticed two small things, beside that everything looks good!

File: styles/designs/cosmos/parts/_keyterms-components.scss (hunk 1 above)
Comment (wilkus27) [reply styles/designs/cosmos/parts/_keyterms-components.scss:20]: Nice catch :wink:

File: styles/output/statistics-pdf.css (hunk 2 above)
Comment (wilkus27) [reply styles/output/statistics-pdf.css:1150]: Thank you for noticing that :wink: It'll be cleaned up in the next PR. Shape that adds margin from below will be removed.

File: styles/designs/cosmos/parts/_keyterms-components.scss (hunk 1 above)
Comment (wilkus27) [reply styles/designs/cosmos/parts/_keyterms-components.scss:20]: It doesn't have an impact on styles, so let me fix that in my second PR for EOC refactoring (https://github.com/openstax/ce-styles/pull/8)

**Review by ajluk — APPROVED** [review]

**Pull request merged**